ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__nutanix-4.16-nightly-x86.yaml

```diff
@@ -0,0 +1,67 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-nutanix-416
+promotion:
+  to:
+  - namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.16"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: control-plane-24nodes
+  cluster: build01
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: nutanix-qe
+    env:
+      COMPUTE_REPLICAS: "24"
+      PROFILE_TYPE: reporting
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-nutanix
+- always_run: false
+  as: data-path-9nodes
+  cluster: build01
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: nutanix-qe
+    env:
```

Comment:
@liqcui For data path test, we use 8 vCPUs 32 GiB size of worker nodes. But I don't see where we can configure the worker node size in ref ipi-conf-nutanix. Could you help to double confirm with installer team?

Reply:
It's in ipi-conf-nutanix-customized-resource-ref.yaml, updated worker node to 8 vCPUs 32 GiB

Reply:
@liqcui Thanks for the info. Then let's also specify the infra node size with SET_ENV_BY_PLATFORM=custom and the below env vars. For infra we used 16 vCPUs 64 GiB
OPENSHIFT_INFRA_NODE_INSTANCE_VCPU=16
OPENSHIFT_INFRA_NODE_INSTANCE_MEMORYSIZE=64Gi

Reply:
do we need that worker cpu and memory size as well as the infra cpu and memory size for all the tests listed here?

Reply:
@paigerube14 @qiliRedHat I have adding new job for 4.17 also, this time I only add above three ENV in datapath job